core/trino-main/src/main/java/io/trino/sql/planner/plan/TableDeleteNode.java

```diff
@@ -68,15 +68,15 @@ public List<PlanNode> getSources()
     }
 
     @Override
-    public <R, C> R accept(PlanVisitor<R, C> visitor, C context)
+    public PlanNode replaceChildren(List<PlanNode> newChildren)
     {
-        return visitor.visitTableDelete(this, context);
+        checkArgument(newChildren.isEmpty(), "newChildren should be empty");
+        return this;
     }
 
     @Override
-    public PlanNode replaceChildren(List<PlanNode> newChildren)
+    public <R, C> R accept(PlanVisitor<R, C> visitor, C context)
     {
-        checkArgument(newChildren.isEmpty(), "newChildren should be empty");
-        return this;
+        return visitor.visitTableDelete(this, context);
```

Comment:
Revert this change. As a far as I can tell, the only thing that changed is the order in which the methods appear.